test/WebSites/FormatterWebSite/Models/AdminValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+
+namespace FormatterWebSite
+{
+    public class AdminValidator : ValidationAttribute
```

Comment:
why use a custom validation attribute?  could instead use any model type with a couple of `[Range]', `[Required]`, ... attributes.